### PR TITLE
feat: use categories from unicode full emoji list

### LINKED ISSUE #445: :frowning face: in under "uncategorised" instead of "people"
Maybe even `:frowning_man:` and `:frowning_woman:`? I only found them because I was looking for them, so I suppose there are even more emoji in the wrong category.

### PR DESCRIPTION
**Pull request by ikatyang**
Fixes #445

Everything is categorized now thanks to [Unicode Full Emoji List](https://unicode.org/emoji/charts/full-emoji-list.html).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,799 +2,1361 @@
...
-| [top](#table-of-contents) | :love_letter: | `:love_letter:` | :bust_in_silhouette: | `:bust_in_silhouette:` |
-| [top](#table-of-contents) | :busts_in_silhouette: | `:busts_in_silhouette:` | :speech_balloon: | `:speech_balloon:` |
-| [top](#table-of-contents) | :thought_balloon: | `:thought_balloon:` | :feelsgood: | `:feelsgood:` |
-| [top](#table-of-contents) | :finnadie: | `:finnadie:` | :goberserk: | `:goberserk:` |
-| [top](#table-of-contents) | :godmode: | `:godmode:` | :hurtrealbad: | `:hurtrealbad:` |
-| [top](#table-of-contents) | :rage1: | `:rage1:` | :rage2: | `:rage2:` |
-| [top](#table-of-contents) | :rage3: | `:rage3:` | :rage4: | `:rage4:` |
-| [top](#table-of-contents) | :suspect: | `:suspect:` | :trollface: | `:trollface:` |
-
-### Nature
```
Comment (mikejpg) [inline README.md]: I

Comment (emmy-dreamchaser) [reply README.md]: nice

File: .travis.yml [hunk 2]
```diff
@@ -1,15 +1,12 @@
 language: node_js
 
 node_js:
-- "8"
+- "12"
```
Comment (mikejpg) [inline .travis.yml]: Cool

File: scripts/generate.js [hunk 3]
```diff
@@ -1,153 +1,12 @@
...
-      } else {
-        reject(
-          error
-            ? error
-            : `Unexpected response status code: ${response.statusCode}`
-        );
-      }
-    });
-  });
+async function generate() {
```
Comment (mikejpg) [inline scripts/generate.js]: Cool

**Comment by Tsatsraa** [comment]
> 
> 
> Fixes #445
> 
> Everything is categorized now thanks to [Unicode Full Emoji List](https://unicode.org/emoji/charts/full-emoji-list.html).

LANTUU

**Comment by Tonmoyhassan666** [comment]
Hii

**Pull request merged**